src/persistency/MySQLStoreAdapter.ts

```diff
@@ -0,0 +1,82 @@
+import { PersistentStoreAdapter, Updater } from './types'
+
+export default class MySQLStoreAdapter implements PersistentStoreAdapter {
+  private dbValidated = false
+
+  constructor(private pool: any, private table: string) {}
+
+  private async validateDB(): Promise<void> {
+    if (!this.dbValidated) {
+      await this.pool.query(
+        `CREATE TABLE IF NOT EXISTS ${this.table} (id varchar(250) primary key, value varchar(500))`,
+      )
+      this.dbValidated = true
+    }
+  }
+
+  public async del(key: string): Promise<void> {
+    await this.validateDB()
+    await this.pool.query(`DELETE FROM ${this.table} WHERE id = ?`, [key])
+  }
+
+  public async get(key: string): Promise<any> {
+    await this.validateDB()
+    const res = await this.pool.query(`SELECT value FROM ${this.table} WHERE id = ?`, [key])
+    const row = res[0][0]
+    return !row || row.length === 0 ? undefined : JSON.parse(row.value)
```

Comment:
minor thing, you can check row using the ? and any positive number is truthy in JS and 0 is falsy so you can write it that way checking the length only:
This should do the same thing but shorter
`return row?.length ? JSON.parse(row.value) : undefined`